lib/features/children/overview/cubit/family_overview_state.dart

```diff
@@ -45,13 +45,13 @@ class FamilyOverviewUpdatedState extends FamilyOverviewState {
   }
 
   // The Givt user profile is first in the list
-  List<Profile> sortedAdults(String givtACcountName) {
+  List<Profile> sortedAdults(String givtAccountID) {
     return adults
       ..sort((a, b) {
-        final compareNames = a.firstName.compareTo(b.firstName);
-        return a.firstName == givtACcountName
+        final compareNames = a.firstName.compareTo(b.id);
+        return a.id == givtAccountID
             ? -1
-            : b.firstName == givtACcountName
+            : b.id == givtAccountID
                 ? 1
                 : compareNames;
       });
```

Comment:
Because of the merge of apps we now have 2 models named Profile that have slight differences, therefore the sorted logic is not shared. I did not dive into making them one profile because that would take more than what we scoped for this issue, but we should at some point.